integration-tests/src/test/resources/performance-features/continuously-create-bridges.feature

```diff
@@ -0,0 +1,130 @@
+                    body:
+                      json:
+                        query: .id
+                        toVar: bridgeId
+            - create-poll:
+              # Wait until the Bridge is in either in ready or failed state
+              - httpRequest:
+                  endpoint: manager
+                  GET: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                    cache-control: no-cache
+                  handler:
+                    body:
+                      json:
+                        query: .status
+                        toVar: bridgeStatus
+              - breakSequence:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: ready
+                  onBreak:
+                    newSequence:
+                      sequence: delete
+                      forceSameIndex: true
+              - fail:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: failed
+                  message: "Bridge creation failed"
+              - thinkTime:
+                  duration: 5s
+              - restartSequence
+            - delete:
+              # Delete Bridge
+              - httpRequest:
+                  endpoint: manager
+                  DELETE: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    authorization: Bearer ${accessToken}
+            - delete-poll:
+              # Wait until the Bridge is in either deleted or in failed state
+              - httpRequest:
+                  endpoint: manager
+                  GET: /api/smartevents_mgmt/v1/bridges/${bridgeId}
+                  headers:
+                    content-type: application/json
+                    authorization: Bearer ${accessToken}
+                    cache-control: no-cache
+                  handler:
+                    status:
+                      store:
+                        toVar: reponseStatus
+                    body:
+                      json:
+                        query: .status
+                        toVar: bridgeStatus
+              - breakSequence:
+                  intCondition:
+                    fromVar: reponseStatus
+                    equalTo: 404
+              - fail:
+                  stringCondition:
+                    fromVar: bridgeStatus
+                    equalTo: failed
+                  message: "Bridge deletion failed"
+              - thinkTime:
+                  duration: 5s
+              - restartSequence
+      """
+    Then run benchmark "rhose-continuously-create-bridges" within 60 minutes
+    And the benchmark run "rhose-continuously-create-bridges" was executed successfully
+    And store Manager metrics in Horreum test "continuously-create-bridges-<users>-users"
```

Comment:
One more idea. Would it be possible to assert the others monitoring metrics in this test? It may be interesting to validate that metrics work properly. Moreover, there will be some new metrics added also to shard operator soon: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1159
What do you think?

Reply:
@jiripetrlik is there a list of new metrics available anywhere?

Reply:
To be honest, I'm not aware of that. I only checked the metrics newly added in https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1159 . However, it is a good question. @manstis are you aware of such list with all metrics? Or should we just try to filter out RHOSE related metrics from /q/metrics endpoint?

Reply:
@sutaakar @jiripetrlik I have [this](https://docs.google.com/document/d/1zHHsrbJVoacZhqZGvr5dIVbGMewCiXZ4Lx85AHaQJnA/edit?usp=sharing) list (let me know if you cannot access it). It relates to [this](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/6e516b4c445e72b0348607ca564f1feab9c4deea/infra/src/main/java/com/redhat/service/smartevents/infra/metrics/MetricsOperation.java) in #1159.

Reply:
@sutaakar I just read this whole PR.. #1159 also adds basic metrics for the `shard-operator`.

Reply:
@manstis thanks

Reply:
@jiripetrlik I don't see any operator metric which would be relevant for tracking to this test.

Reply:
@sutaakar Ok, understand. It was just an idea.